GravityPDF/gravity-pdf
# Use the ID `tab_PDF` for both global and form settings pages

## Description
Previously, the global settings used `tab_PDF` and the form settings uses `tab_pdf`. This PR consolidates the two into `tab_PDF`. 

## Testing instructions
1. View Forms -> Settings -> PDF
2. View Forms -> Form -> Hover Form -> PDF

Verify both pages work as expected

## Additional Comments 

This reduces our CSS markup, and just generally makes things easier.
